### PR TITLE
Fix Some Typos for Bell Canada Training Lab Guides

### PR DESCRIPTION
**Pull request by rjamal-av**
Fix Some Typos for Bell Canada Training Lab Guides

### CODE REVIEW

**Comment by rjamal-av** [comment]
Fix some typos for Bell Canada Training  Lab Guides

**Review by jb-smoker — APPROVED** [review]
Lgtm. Will await @joe-amendolara 's approval before merge since it was requested.

**Review by joe-amendolara — APPROVED** [review]
approved!

**Pull request merged**
